scripts/install-include-headers.sh

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env bash
+#
+# Install all the necessary necessary include files from wasi-libc
+# (`$WASI_LIBC`) into the sysroot include directory (`$SYSROOT_INC`). It uses a
+# passed target triple (`$TARGET_TRIPLE`) to condition some of the
+# copied/generated files.
+#
+# Usage: SYSROOT_INC=... TARGET_TRIPLE=... ./install-include-headers.sh
+
+set -e
```

Comment:
I recommend adding `set -euo pipefail` to all bash scripts. In particular here, `-u` will catch the case where someone forgets to set `SYSROOT_INC` or `TARGET_TRIPLE`.